Cargo.toml

```diff
@@ -19,7 +19,7 @@ http = "0.2.6"
 pulldown-cmark = { version = "0.9.1", default-features = false }
 serde = { version = "1.0.136", features = ["derive"] }
 serde_json = "1.0.79"
-spin-sdk = { git = "https://github.com/fermyon/spin", rev = "139c40967a75dbdd5d4da2e626d24e68f54c0a5a", optional = true }
+spin-sdk = { git = "https://github.com/fermyon/spin", optional = true }
```

Comment:
Strongly recommend `branch` or `tag` so you're not build off a moving target.

Reply:
Good point. I will tag it to `v0.9.0`